scripts/data-lint.js

```diff
@@ -0,0 +1,50 @@
+const inspect = require("util").inspect;
+const { join, resolve } = require("path");
+
+const {
+	communitySchema,
+	demosSchema,
+	pluginsSchema,
+	sitesSchema,
+	startersSchema
+} = require("./schemas");
+
+const { schemaLint } = require("./data-lint-lib");
+
+const dataDir = resolve(join(__dirname, "..", "src", "_data"));
+let errors = [];
+
+main();
+
+async function main() {
+	await Promise.all([
+		dataLint(join(dataDir, "community"), "*.js", communitySchema),
+		dataLint(join(dataDir, "demos"), "*.js", demosSchema),
+		dataLint(join(dataDir, "plugins"), "*.json", pluginsSchema),
+		dataLint(join(dataDir, "sites"), "*.json", sitesSchema),
+		dataLint(join(dataDir, "starters"), "*.json", startersSchema)
+	]);
+
+	if (errors.length) {
+		console.error(inspect(errors, { depth: 4 }));
```

Comment:
How does the `4` came to be here?

Reply:
How deep to recurse into objects.
https://nodejs.org/api/util.html#util_util_inspect_object_showhidden_depth_colors

> `depth` &lt;number&gt; Specifies the number of times to recurse while formatting `object`. This is useful for inspecting large objects. To recurse up to the maximum call stack size pass `Infinity` or `null`. **Default: 2.**

We could switch to `JSON.stringify()` if you prefer.

Reply:
No, it's fine. I just have seen this „magic number” and thought, whether it has some special requirement, so should be documented.
Why do you want to recurse four steps deep?
Is it related to Ajv? The schema?
You must have had a reason to differ from the default.

Reply:
I think the default was `2`, which meant that i was just getting random `[Array]` or `[Object]` output instead of the actual nested output I wanted.

Reply:
Okay, so would you be okay with me adding a constant to „unpack objects” or something instead of the `4`?

Reply:
Yeah, whatever you think is best. 

Another possible interesting point is that `util.inspect` just returns a string that looks like some stuff. It isn't valid JSON or anything consumable, just a debugging string. If you want to switch to `JSON.stringify(errors, null, 2)`, it would output some valid JSON that you could pipe to a file or otherwise manipulate.